presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -273,6 +273,8 @@ primaryExpression
     | POSITION '(' valueExpression IN valueExpression ')'                            #position
     | '(' expression (',' expression)+ ')'                                           #rowConstructor
     | ROW '(' expression (',' expression)* ')'                                       #rowConstructor
+    | qualifiedName '(' ASTERISK ')' filter?                                         #functionCall
```

Comment:
This isn't correct, since filter can be used with a window function. It should be added to the existing function call lines below.

Even if we don't support executing this today, we should parse it, and we need handling for it in the analyzer since the AST can represent it.